.github/workflows/dist.yml

```diff
@@ -0,0 +1,71 @@
+# when `latest` tag or another tag starting with `v` is pushed,
+# this workflow runs to push the distrubtion into a `dist-<tag>`
+# tag
+name: DISTRIBUTE
+
+on:
+  push:
+    tags:
+      - latest
+      - v*
+
+jobs:
+  distribute:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: "checkout"
+        uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      # always test before dist
+      - uses: eskatos/gradle-command-action@v1
+        name: Build and Test
+        with: 
+          build-root-directory: AndroidXCI
+          wrapper-directory: AndroidXCI
+          wrapper-cache-enabled: true
+          dependencies-cache-enabled: true
+          configuration-cache-enabled: true
+          arguments: check :ftlModelBuilder:check
+      # build distribution
+      - uses: eskatos/gradle-command-action@v1
+        name: Build Distribution
+        with: 
+          build-root-directory: AndroidXCI
+          wrapper-directory: AndroidXCI
+          wrapper-cache-enabled: true
+          dependencies-cache-enabled: true
+          configuration-cache-enabled: true
+          arguments: :cli:installDist
+      # push it to a new git tag
+      - name: "create dist branch"
+        id: event-args
+        run: |
+          set -x
+          REF="${{ github.event.ref }}"
+          TAG=$(echo $REF| sed 's/refs\/tags\///')
+          DIST_TAG="dist-$TAG"
+          git config user.email "${{ github.event.head_commit.author.email }}"
+          git config user.name "${{ github.event.head_commit.author.name }}"
```

Comment:
Is this part needed? It's also not necessarily the person who triggered the release anyway right? WDYT about just leaving the git author to default so we can see it was pushed via this action and it'll also make this script shorter?

Reply:
i had to add it, i don't think checkout action sets it.

I don't think we can get the author of the tag but tip of tree author seems fine?
I'm ok to use another name as well (does not really simplify much though.)

Reply:
Ack